src/jdk.jartool/share/classes/sun/tools/jar/GNUStyleOptions.java

```diff
@@ -228,6 +228,20 @@ void process(Main jartool, String opt, String arg) {
                 }
                 boolean isHidden() { return true; }
             },
+            new Option(true, OptionType.OTHER, "--release") {
+                void process(Main jartool, String opt, String arg) throws BadArgs {
```

Comment:
This should now have an additional validation check that this `--release` option has been used only when `--describe-module` or `--validate` option has been specified. I realize that this proposed change currently does a check for these two main operations before invoking this option processing, but I think we should also include the main operation check here and throw  option processing failure if the main operations aren't the ones we expect.